packages/components/src/DateTimePickers/Date/Manager/Manager.component.js

```diff
@@ -56,7 +63,7 @@ function ContextualManager(props) {
 			value={{
 				value: {
 					textInput: state.textInput,
-					date: state.date,
+					date: state.localDate,
```

Comment:
`state.localDate` stores date in local timezone,
`state.date` stores date in target timezone.